CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## 2.16.0-dev.0
+## 2.17.0
+* Revert perfetto bundle [#4422](https://github.com/flutter/devtools/pull/4422)
```

Comment:
can you also add the entry for this PR manually?
`* Prepare for 2.17.0 release [#4428](https://github.com/flutter/devtools/pull/4428)`